process/banning.html

```diff
@@ -44,37 +44,55 @@ <h4>Nearby</h4>
         </li>
       </ul>
     </div>
-    
+
     <div id="Content">
 
 <p>The W3C process document empowers the Director to <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">suspend or remove participants from groups</a>. This document provides details about implementation and delegation.</p>
 
-<p>1. Consistent with the practices below, the Director delegates authority to suspend or remove participants:</p>
+<ol>
+<li>
+<p>Consistent with the practices below, the Director delegates authority to suspend or remove participants:</p>
 <ul>
 <li>from a single group, to the CEO;</li>
 <li>from a single group that the individual has not formally joined, to the chair(s) of the group; and</li>
 <li>from all W3C groups, to the CEO and COO jointly.</li>
 </ul>
 <p>If the chair, team contact, CEO, COO is the one accused of unacceptable behavior, they should not be involved in the decision to ban.</p>
-
-<p>2. Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, chairs may temporarily or permanently suspend participation by a person who has not formally joined the group.</p>
-
-<p>3. A W3C group chair and team contact may propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO.</p>
+</li>
+<li><p>In case of disagreement between the W3C group chair and team contact, each should explain their reasons to the CEO.</p></li>
+<li>
+<p>Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code
+of Ethics and Professional Conduct</a>, a W3C group chair and team contact should
+give at least one written warning. If the person is a is a W3C Member representative
+participant in the Group, the Member's Advisory Committee representative must be
```

Comment:
"Member representative participant" reads awkwardly.  I suggest "If the participant was appointed by a Member, the Member's Advisory ..."